recipes/dd-agent.rb

```diff
@@ -137,3 +137,5 @@ def template_vars
 
 # Install integration packages
 include_recipe 'datadog::integrations' unless is_windows
+
+include_recipe 'datadog::system-probe'
```

Comment:
perhaps we can do the equivalent of: `unless system_probe_binary_doesnt_exist`?

Reply:
`is_windows` is another safe thing to include, though that would be covered by the `system_probe_binary_doesnt_exist` value

Reply:
This is unfinished work, it’s not totally working yet, I’ll finish up tomorrow